### PR TITLE
release-v1.1

### PR DESCRIPTION
**Pull request by ypf79**
release v1.1

### CODE REVIEW

File: adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc [hunk 1]
```diff
@@ -1,431 +1,1167 @@
...
+* *`li.d` :* 
 
-.4+<.^|*`li.d`* *`$rd`*, *`imm64`*
+[source]
+----
+li.d                $t0, 0xfedcba9876543210
+Machine Instruction ------------------------------------------------------------------
+lu12i.w             $rd, si20         // load IMM64[31:12]    $t0 = 0x0000000076543000
+ori                 $rd, $rd, si12    // load IMM64[11: 0]    $t0 = 0x0000000076543210
+lu32i.d             $rd, si20         // load IMM64[51:32]    $t0 = 0x000cba9876543210
```
Comment (xen0n) [inline adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc:106]: Isn't sign extension performed here?

```suggestion
lu32i.d             $rd, si20         // load IMM64[51:32]    $t0 = 0xfffcba9876543210
```

File: adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc [hunk 2]
```diff
@@ -1,431 +1,1167 @@
...
 
 [options="header"]
-[cols="40,10,50"]
+[cols="90,10"]
+|===========================
+^.^|opcode
+^.^|src1
+
+^.^|*`jr`*
+^.^|*`$rd`*
```
Comment (xen0n) [inline adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc:125]: jr's operand is input-only and it's rj in the realized instruction.

```suggestion
^.^|*`$rj`* 
```

File: adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc [hunk 3]
```diff
@@ -1,431 +1,1167 @@
...
+|===========================
 
-<.^|*`pcalau12i`* *`$rd`*, *`%got_pc_hi20`*(*`sym`*)
-<.^|*`ld.d`* *`$rd`*, *`$rd`*, *`%got_pc_lo12`*(*`sym`*)
+*Usage :* 
 
-.2+^.^|*`GTOPCR`*
+[source]
+----
+jirl                $t4
```
Comment (xen0n) [inline adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc:144]:
```suggestion
jr                  $t4
```

File: adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc [hunk 4]
```diff
@@ -1,431 +1,1167 @@
...
-.6+<.^|*`la.local`* *`$rd`*, *`sym_local`*
+*Usage :* 
 
-.2+^.^|*`NORMAL`*
+[source]
+----
+tail36              $t1, sym_tail
+Machine Instruction ----------------------
+pcaddu18i           $t1, %call36(sym_tail)
+jirl                $t1, $t1, 0
```
Comment (xen0n) [inline adoc/CHAPTER-9.Pseudo_Instruction_Set/9.Pseudo_Instruction_Set.adoc:264]: Tail calls do not write to any link register.

```suggestion
jirl                $zero, $t1, 0     
```

File: README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ This is the official documentation of the Assembly Language Programming Guide fo
 
 ## Releases
 
-The latest Assembly Language Programming Guide documentation releases are available at https://github.com/loongson/la-asm-manual/releases and are licensed under the Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0) License.
+The latest Assembly Language Programming Guide documentation releases are available at https://github.com/loongson/la-asm-manual/releases and are licensed under the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) License.
```
Comment (xen0n) [inline README.md:7]: 👀

(I would still strongly recommend the Loongson legal team to reconsider the license choices, because even PRs would become infringement with an CC-ND license in place.)

**Review by xen0n — COMMENTED** [review]
Just a casual drive-by review with a few comments.

File: README.md (hunk 5 above)
Comment (SixWeining) [reply README.md:7]: Forward a message from our intellectual property department:

I hope this message finds you well.
I am writing to provide some clarification regarding contributions to the project through Pull Requests (PRs) and how it relates to the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) license.
It has come to my attention that there might be some confusion about the applicability of the CC BY-NC-ND 4.0 license to contributors who wish to submit changes to the documentation via PR. I would like to reassure you that as a contributor, you are indeed permitted to make submissions through PR, and the CC license conditions primarily pertain to downstream recipients, not to the contributors themselves.
The CC BY-NC-ND 4.0 license allows for the sharing and use of the material under certain conditions, which are primarily directed at those who receive and use the material, ensuring that they give appropriate credit, do not use the material for commercial purposes, and do not create adaptations of the material. However, the act of contributing to the project, especially when explicitly welcomed by the project's contribution policy, is considered a separate matter and is typically governed by the terms set forth in the project's contribution guidelines.
When you contribute to the project through a PR, you are not considered a downstream recipient under the CC BY-NC-ND 4.0 license; instead, you are an active participant in the collaborative development process. Your contributions are subject to the project's contribution policy, which may include additional terms and conditions for submission, review, and merging of changes.
In summary, as long as you adhere to the project's contribution guidelines and respect the spirit of the CC BY-NC-ND 4.0 license, your contributions via PR are both welcomed and permissible. If you have any further questions or concerns, please do not hesitate to reach out.
Thank you for your understanding and for your valuable contributions to the project.
Best regards,
